bundles/org.openhab.binding.modbus.stiebeleltron/README.md

```diff
@@ -0,0 +1,205 @@
+# Stiebel Eltron ISG Binding
+
+This extension adds support for the Stiebel Eltron modbus protocol.
+
+An Internet Service Gateway (ISG) with an installed modbus extension is required in order to run this binding. In case the modbus extension is not yet installed on the ISG, the ISG Updater Tool for the update can be found here: https://www.stiebel-eltron.de/de/home/produkte-loesungen/erneuerbare_energien/regelung_energiemanagement/internet_servicegateway/isg_web/downloads.html
```

Comment:
Can you put each line on a new line so it makes it easier to review (future) improvements to the documentation.

Reply:
It think I implemented your feedback. Please have a look and let me know in case I didn't understand your review comment properly.